processor/runtime/src/main/java/com/redhat/service/smartevents/processor/errorhandler/KafkaErrorPublisher.java

```diff
@@ -0,0 +1,48 @@
+package com.redhat.service.smartevents.processor.errorhandler;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.reactive.messaging.Outgoing;
+import org.reactivestreams.Publisher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.smallrye.mutiny.operators.multi.processors.BroadcastProcessor;
+
+@ApplicationScoped
+public class KafkaErrorPublisher implements ErrorPublisher {
```

Comment:
This puts back-channel meta-messages on the Kafka topic to be consumed by our Sidecar [*]...

[*] We'd intended to use a Sidecar on KNative but @danielezonca is pushing for us to use a Source Processor defined, in part, on the create Bridge API. Research and refactoring is on going...

Reply:
Deleted with the commit to use Kafka's DLQ support.